src/sage/rings/ring.pyx

```diff
@@ -1474,24 +1485,11 @@ cdef class Field(CommutativeRing):
 
 
 cdef class Algebra(Ring):
-    """
-    Generic algebra
-    """
-    def __init__(self, base_ring, names=None, normalize=True, category=None):
-        """
-        Initialize ``self``.
-
-        EXAMPLES::
+    _default_category = _Rings
```

Comment:
Does this not break user code, as there's nothing that sets the correct category?

Reply:
better now